events/minecraft/message.js

```diff
@@ -146,6 +146,32 @@ module.exports = {
                     files: [new MessageAttachment(generateMessageImage(formattedMessage), 'guildEvent.png')],
                 });
             }
+            if (config.webhook.enabled) {
+                const event = determineEvent(msgString);
+                if (Object.keys(config.webhook.events).includes(event)) {
+                    if (config.webhook.events[event]) {
```

Comment:
isn't this just the same if statement twice?

Reply:
Now that I think of it, it might be yeah